lib/main_common.pm

```diff
@@ -328,12 +328,16 @@ sub is_using_system_role {
     #system_role selection during installation was added as a new feature since sles12sp2
     #so system_role.pm should be loaded for all tests that actually install to versions over sles12sp2
     #no matter with or without INSTALL_TO_OTHERS tag
-    return is_sle('>=12-SP2')
+    # On SLE 15 SP0 we unconditionally have system roles screen
+    # SLE 15 SP1 has system roles only if more than one is available, meaning either registered or with all packages DVD
+    # On kubik, leap 15.1+, TW we have it instead of desktop selection screen
```

Comment:
s/kubik/kubic/ :)

Reply:
Sorry, seems that I've started to use german way of writing =) The band 'kreator' got it's name in the similar way =P